daemon/mgr/spec_nvidia_hook_test.go

```diff
@@ -0,0 +1,114 @@
+package mgr
+
+import (
+	"os"
+	"os/exec"
+	"path"
+	"reflect"
+	"testing"
+
+	"github.com/alibaba/pouch/apis/types"
+	"github.com/opencontainers/runtime-spec/specs-go"
+)
+
+func Test_setNvidiaHook(t *testing.T) {
+	nvidiaHookName = "test-nvidia-container-runtime-hook"
+	installDir := "/usr/local/bin/"
+	fullname := path.Join(installDir, nvidiaHookName)
+	os.Remove(fullname)
+	os.Create(fullname)
+	os.Chmod(fullname, 0755)
```

Comment:
How about adding `defer os.Remove(fullname)` here?  delete the `test-nvidia-container-runtime-hook` after test finished

Reply:
Actually, there is defer function at line22. 
Sometimes test was broken unexpected, we should delete the mock file at first.

Reply:
Take it easy man 😁